src/pattern-library/components/patterns/ContainerPatterns.js

```diff
@@ -1,78 +1,105 @@
 import { useState } from 'preact/hooks';
 
-import {
-  PatternExample,
-  PatternExamples,
-  PatternPage,
-  Pattern,
-} from '../PatternPage';
+import Library from '../Library';
 
 import { IconButton, LabeledButton } from '../../../';
 
 export default function ContainerPatterns() {
   const [showModalExample, setShowModalExample] = useState(false);
   return (
-    <PatternPage title="Containers">
-      <Pattern title="Frame">
-        <p>
-          A <code>frame</code> has a border, background, padding, and vertical
-          spacing of immediate children.
-        </p>
-        <PatternExamples>
-          <PatternExample details="basic frame">
-            <div className="hyp-frame">This is in a frame.</div>
-          </PatternExample>
-          <PatternExample details="child content in frame">
+    <Library.Page title="Containers">
+      <Library.Pattern title="Frame">
+        <Library.Example title="Basic frame">
+          <p>
+            A frame provides a border, background, padding and vertical spacing
+            of immediate children.
+          </p>
+          <Library.Demo withSource>
             <div className="hyp-frame">
-              <div className="hyp-u-border">Child content in a frame.</div>
-              <div className="hyp-u-border">Child content in a frame.</div>
-              <div className="hyp-u-border">Child content in a frame.</div>
+              <div>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
```

Comment:
I'm using lorem ipsum in some of these examples, but the meat of the examples have not changed.

Reply:
Do you have a `Lorem ipsum` generator integrated into your IDE? Otherwise, maybe we should have one as a utility.